src/tests/api/session.test.ts

```diff
@@ -0,0 +1,67 @@
+import { assert200OK, assert400BadRequest, assert401Unauthorized } from "../../utils/apiCommonAssertions";
+import { sessionPost, stubStartPost } from "../../utils/apiTestSteps";
+
+describe("Happy Path Get SessionID", () => {
+	let stubResponse: any;
+
+	beforeEach( async () =>{
+		const requestBody = {
+			target:"https://unb2uhs7rl.execute-api.eu-west-2.amazonaws.com/dev/",
+		};
+		stubResponse = await stubStartPost(requestBody);
+
+	});
+    
+	it("Given valid endpoint has been received", async () => {
+
+		const response = await sessionPost(stubResponse.data.clientId, stubResponse.data.request);
+
+		assert200OK({ status: response.status, statusText: response.statusText });
```

Comment:
We should check a session Id is returned.

Reply:
Session id is validated in every happy path test that uses this value